lib/packwerk/inflector.rb

```diff
@@ -28,17 +28,11 @@ def from_file(inflections_file)
         custom_inflector: Inflections::Custom
       ).void
     end
-    def initialize(custom_inflector: Inflections::Custom.new)
-      @custom_inflector = custom_inflector
-      @default_inflector = Inflections::Default
+    def initialize(custom_inflector:)
```

Comment:
This change to the `custom_inflector` parameter is in the wrong commit. It belongs in “Make custom_inflector argument in Inflector required”.